rpcs3/Loader/PSF.h

```diff
@@ -108,4 +108,10 @@ namespace psf
 	{
 		return {format::array, max_size, value};
 	}
+
+	// Checks if of HDD catgeory (assumes a valid category is being passed)
+	constexpr bool is_cat_hdd(std::string_view cat)
+	{
+		return cat[1] != 'D' && cat != "DG"sv && cat != "MS"sv;
```

Comment:
what if cat size is < 2 ?

Reply:
I don't care if it should be fine, it's standard procedure to check for such things

Reply:
Okay.